### ISSUE TITLE
Suggest removing 'pending' for now

### ISSUE THREAD

**Issue by michaelcfanning** [sarif-standard/sarif-spec#53]
Investigations with a likely tool to provide this class of issue shows that the use pattern is not clear. We have also not defined an explicit way to provide the relevant result data for post-processing. The 'formattedMessage' result looks like (as it has a notion of message args) but requires the use to opt into the formatted message mechanism.

The binary inspection tool I examined (https://github.com/microsoft/binskim) has a data-driven check that looks for linkage to vulnerable dependencies. The list of dependencies comprises the policy for the rule. In the absence of any policy whatsoever, the tool would be required to emit every dependency it locates as a 'pending' message, which greatly increases the size of the log file. Another check is even worse, it ensures that a certain kind of function-level security has not been disabled. The check includes a small 'allow list' of well-known (safe) functions with the insecure declaration. In the absence of explicit policy, the rule would need to emit a 'pending' message for all functions encountered,

Instead of taking the 'pending' approach, binskim fires a single warning at analysis time, per policy-driven rule, that notes the check has been disabled because policy has not been configured. The user is prompted to emit a policy file (the tool can produce a default one via a separate command) or to provide an explicit command line value that indicates the analysis should use the current built-in default policy.

Besides the log file size issues, we have no clear way for tools to emit the values that would be subsequently evaluated. Options that exist, assuming the current format, are to have advance knowledge of the message output and to parse it, or to force tools to emit a formattedMessage object (which has a notion of message args that separate the dynamic values associated with a result).

Long story short, let's remove the 'pending' issue type for now until we have a clearer sense of a practical pattern for use.
